src/main/java/stsjorbsmod/powers/WitherPower.java

```diff
@@ -0,0 +1,31 @@
+package stsjorbsmod.powers;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.DamageAllEnemiesAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+
+public class WitherPower extends CustomJorbsModPower {
+    public static final StaticPowerInfo STATIC = StaticPowerInfo.Load(WitherPower.class);
+    public static final String POWER_ID = STATIC.ID;
+    public WitherPower(AbstractCreature owner) {
+        super(STATIC);
+        this.amount = -1;
+        this.updateDescription();
+        this.owner = owner;
+        this.type = PowerType.DEBUFF;
+        this.img = this.region48.getTexture();
```

Comment:
This shouldn't be necessary (should be covered by the `CustomJorbsModPower` ctor)